src/main/java/com/checkmarx/flow/dto/iast/manager/dto/description/VulnerabilityDescription.java

```diff
@@ -0,0 +1,39 @@
+package com.checkmarx.flow.dto.iast.manager.dto.description;
```

Comment:
why you need new vulnerability description?

Reply:
I used that information from Iast. Don't understand a question. that is a DTO object which I get from IAST.